src/main/java/com/quizapp/enterprise/models/User.java

```diff
@@ -1,18 +1,26 @@
 package com.quizapp.enterprise.models;
 
-import lombok.Data;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.Hibernate;
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
+import java.util.Objects;
 
 @Entity
 @Table(name = "users")
-@Data
```

Comment:
According to many sources including the IntelliJ JPA plugin, using @Data on JPA classes can lead to many bad side-effects

https://dzone.com/articles/lombok-and-jpa-what-may-go-wrong